.github/workflows/ecr-build-push.yml

```
@@ -0,0 +1,153 @@
+    needs: build-and-push
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop')
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::025434362120:role/github-actions-role
+          source-account-id: '025434362120'
```

actionlint (1.7.8)

input "source-account-id" is not defined in action "aws-actions/configure-aws-credentials@v4". available inputs are "audience", "aws-access-key-id", "aws-region", "aws-secret-access-key", "aws-session-token", "disable-retry", "http-proxy", "inline-session-policy", "managed-session-policies", "mask-aws-account-id", "output-credentials", "output-env-credentials", "retry-max-attempts", "role-chaining", "role-duration-seconds", "role-external-id", "role-session-name", "role-skip-session-tagging", "role-to-assume", "special-characters-workaround", "unset-current-credentials", "use-existing-credentials", "web-identity-token-file"

(action)